native/cocos/physics/physx/shapes/PhysXCone.cpp

```diff
@@ -33,9 +33,12 @@
 namespace cc {
 namespace physics {
 
-PhysXCone::PhysXCone() : _mMesh(nullptr){};
+PhysXCone::PhysXCone() : _mMesh(nullptr),
+                         PhysXShape(){};
```

Comment:
- `;` is not needed
- PhysXShape is base class of `PhysXCone `? Even so, i don't think should invoke the constructor explicitly.